core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeLexer.g4

```diff
@@ -941,6 +941,8 @@ SQLCOMMAND:
     ) ~[\r\n]*
 ;
 
+ANY_BANG: BANG ID ~[\r\n]*;
```

Comment:
remove from this PR, it's irrelevant to the description of PR.

@jimidle , please take a look at this separately. is there a better way of doing `SQLCOMMAND`, so that `ANY_BANG` won't interfere with it?

Reply:
Reverted will raise a seperate PR